dataland-frontend/tests/e2e/specs/sfdr/index.ts

```diff
@@ -0,0 +1,6 @@
+/**
```

Comment:
The test currently does not run in the CI because it is not included in the master level index.ts

Reply:
Since this has happened several times in the past, how about we set a reminder in the PR-template?

Something like:
"If you have created new test files, make sure that they are included in a test container and actually run in the CI"

Reply:
I think i've added the entry  require("./sfdr"); to the index.ts file.
It is located after the  require("./lksg");

Reply:
I think github just did not refreshed the page. But i like your suggestion to add a reminder to the template

Reply:
I have updated the pr template